### PR TITLE
Aliaslist to pms

### PR DESCRIPTION
**Pull request by ddday**
A solution to #219 , instead of !aliaslist showing up in the chat, it will show up in your pms.

### CODE REVIEW

File: cogs/alias.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def __init__(self, bot):
             fileIO("data/alias/aliases.json", "save", self.aliases)
         await self.bot.say("Alias '{}' deleted.".format(command))
 
-    @commands.command(pass_context=True)
+    @commands.command(pass_context=True, no_pm=True)
```
Comment (irdumbs) [inline cogs/alias.py:86]: nice catch

**Comment by tekulvw** [comment]
Two things: Line 93 you didn't change `bot.say` to `bot.whisper`, and in #195 I wanted to put `aliaslist` into the `alias` group. Thoughts on that? Because if that's something we agree is wanted then I'll just make these changes in that PR

**Comment by ddday** [comment]
I did change it to bot.whisper, maybe I just did the PR wrong, otherwise it wouldn't work(which it does). Also, sure, I guess that would be fine.

**Comment by tekulvw** [comment]
Added with 7e4c41fe

**Pull request closed without merging**